packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -88,76 +88,66 @@ export function useDefaultViews( { postType } ) {
 				title: __( 'Published' ),
 				slug: 'published',
 				icon: published,
-				view: {
-					...DEFAULT_POST_BASE,
-					filters: [
-						{
-							field: 'status',
-							operator: OPERATOR_IS_ANY,
-							value: 'publish',
-						},
-					],
-				},
+				view: DEFAULT_POST_BASE,
+				filterDataBy: [
```

Comment:
Does this really need to be in a "filters" format? I mean this can be just `status: publish` and inject it directly into the REST API query. Nothing forces us to use the dataviews format here.

The only reason the dataviews format would be good here is if we consider default views just like custom views, views that are saved in the backend and that the user can edit... But since we don't want the user to be able to edit these, it might not be worth the added complexity of using this format. 

Anyway, If you feel otherwise, that's ok. It's not a big deal for me.

Reply:
We have to do two things here:

1. Applying this to the REST API => your suggestion makes a lot of sense and simplifies things.
2. Applying this to fields, so they don't have filters that are already declare here => If we go with the REST API nomenclature here, we'd have to create another mapping when fields are created. For example, it can be `author` or `author_exclude` in the REST API but the field id is always `author`. I just wanted to avoid creating yet another mapping by leveraging a nomenclature that users are already familiar with.

No strong opinions, can do whatever folks prefer.

Reply:
> Applying this to fields, so they don't have filters that are already declare here 

Can you clarify this point. Is it about hiding the field from the available filters?

Reply:
Ok, so If I'm not wrong, we want to hide the "status" from available filters in all the views except "all pages".

First, that we need a dedicated flag on the field object to do that. It shouldn't be the absence of "elements" that triggers that.

Now, it's true that we have two ways of looking at this "default views". 

 - if they are equivalent to custom views, your approach in the first PR makes sense assuming that we want to hide specific filters values. But I would argue that if these filters can be any random filters, it's bad to hide the filters because we don't really know what they represent. Maybe that filter say "title that start with a" (random filter). In that case, why I shouldn't be able to add another filter as a user that says "title contain b". So in this case hiding the filters doesn't really make sense.

  - The custom views are not "real views", they are just random WP-Admin pages. In this case, there's no need to have a "filters" key, we can just say `status: "publish"` and add `enableFilter: false` to the "status" field in all pages but the "all pages".

Does that reasoning make sense?

--- 

Anyway, I shared my reasoning but I don't think it's that important for now. I'm happy to approve this PR as no new APIs or anything. That said I'd prefer if we keep "filters" as key if we go with the current approach instead of `filterDataBy`.

Reply:
> Does that reasoning make sense?

That's in line with what I shared in the [other PR](https://github.com/WordPress/gutenberg/pull/65264#issuecomment-2346052895). Essentially, we need to figure out how something like this can work in custom views across permission levels (or same user). Custom views are just a view config right now, so if we don't want this to be part of the view config like in 65264, we'd need to make custom views a different thing that can declare changes to view, fields, and datasets (depending on implementation).

> That said I'd prefer if we keep "filters" as key if we go with the current approach instead of filterDataBy.

Done at [cb9b210](https://github.com/WordPress/gutenberg/pull/65295/commits/cb9b2102eb2f5f51a4be7c1108846fda2ed0b68f)